src/components/ceremony/ceremony.repository.ts

```diff
@@ -32,15 +32,20 @@ export class CeremonyRepository extends DtoRepository<
       actualDate: input.actualDate,
       canDelete: true,
     };
-    return await this.db
+    const result = await this.db
       .query()
       .apply(await createNode(Ceremony, { initialProps }))
-      .return<{ id: ID }>('node.id as id')
+      .return<{ node: UnsecuredDto<Ceremony> }>('node')
```

Comment:
That's not what this does. I'd revert this. The object with single ID is sufficient.

Reply:
Right but aren't we trying to return the entire entity now vs just an ID? I feel like I am going in circles on everything and don't have a clear idea on what the expectations are for query implementations

Reply:
Well generally we are. Because this object isn't created directly by users though, the returned object isn't really used. Only the ID. Keeping the return as `{ id: ID }` is best as that allows the edgedb implementation to do that plus more without it being a breaking type.